### ISSUE TITLE
weight-only

### ISSUE THREAD

**Issue by zhouxin666251** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#196]
if i want use the chpeckpoint training method. if weight-only set to True or False affect the training results greatly? I found if set weight-only default=Fasle the loss of heatmps and pafs will biger.

**Comment by Daniil-Osokin** [comment]
Hi! Model weights are used to give better initialization when training starts, than random initialization. That is why there is `--weights-only` option. Another thing is that if you have stopped the training at some time (e.g. electricity was switched off suddenly, or a noise from PC disturbs your relatives while they are slipping), it is better not to loose current trained model, but save it and continue training next time from this saved state keeping all previous progress. That is why there is `--checkpoint-path` option. You can check these tutorials for the details:
* [Saving and loading a general checkpoint in PyTorch](https://pytorch.org/tutorials/recipes/recipes/saving_and_loading_a_general_checkpoint.html).
* [Saving and loading models](https://pytorch.org/tutorials/beginner/saving_loading_models.html).

BTW, I see you have a lot of questions. If you are doing a particular project, we can discuss the details online to not spend time on writing.

**Comment by zhouxin666251** [comment]
Thans a lot,  i will take your advice
